<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryForn</class>
 <widget class="QWidget" name="HistoryForn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>History Transaksi</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(158, 158, 158);</string>
  </property>
  <widget class="QPushButton" name="HomeButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/arza0/Downloads/back.png</normaloff>C:/Users/arza0/Downloads/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TOP-UP</string>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_topup">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="currentText">
    <string>10000</string>
   </property>
   <item>
    <property name="text">
     <string>10000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100000</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>59</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Saldo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
